From c82a993548494ee13035ca77aeaf0726218bee0d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Tue, 15 Jun 2021 23:25:03 +0200
Subject: [PATCH] Do not install top-level "tests" package

Fixes a regression in af2f16335279044e8baeab42e0b182d3e05861b4 that
causes "tests" to be installed as a top-level package to site-packages.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index b4aef13..afbe57a 100644
--- a/setup.py
+++ b/setup.py
@@ -13,7 +13,7 @@ setup(
     description='Compress responses in your Flask app with gzip, deflate or brotli.',
     long_description=LONG_DESCRIPTION,
     long_description_content_type='text/markdown',
-    packages=find_packages(),
+    packages=find_packages(exclude=['tests']),
     zip_safe=False,
     include_package_data=True,
     platforms='any',
-- 
2.32.0

